### PR TITLE
add forbiddenApis plugin and add locale to String.format calls

### PR DESCRIPTION
**Pull request by mfussenegger**

### CODE REVIEW

**Comment by seut** [comment]
travis fails with compilation errors

File: sql/build.gradle [hunk 1]
```diff
@@ -1,5 +1,6 @@
 apply plugin: 'idea'
 apply plugin: 'java'
+apply plugin: 'de.thetaphi.forbiddenapis'
```
Comment (seut) [inline sql/build.gradle]: would be nice to enable it on every java sub-project, specially if you already started to add missing locales on them (afaik you did at blob, core, dns-discovery)

**Comment by seut** [comment]
otherwise ack

File: sql/build.gradle (hunk 1 above)
Comment (mfussenegger) [inline sql/build.gradle]: I'll do a follow up PR for the other modules because there are more errors to be fixed.

**Pull request merged**